layouts/index.html

```diff
@@ -29,10 +29,12 @@ <h2>{{ .Site.Params.Description }}</h2>
 </div>
 {{ end }}
 
-{{ if os.FileExists "index-about.md" }}
-<div class="markdown-content">
-    {{ readFile "index-about.md" | markdownify }}
-</div>
+{{ if (or
+    (and (fileExists "content/_index.md") .Content)
+    (fileExists "content/index-about.md")) }}
+        <section class="home-content">
+            {{ or .Content (readFile "index-about.md" | markdownify) }}
```

Comment:
@jamesericdavidson Shouldn't the readFile be whichever file it checks ?

Reply:
@yashmehrotra If I understand you correctly, `readFile` is not needed for `_index.md`, as that file supports the `{{ .Content }}` method.

If `_index.md` exists (and has content), `index-about.md` isn't read.

This de-facto deprecates `index-about.md`, while avoiding breakage (i.e. forcing the user to `mv index-about.md _index.md`)

Reply:
`{{ .Content }}` also supports hot reloading, whereas `readFile` doesn't - you need to restart the local server to observe changes.